ios/fastlane/metadata/de-DE/name.txt

```diff
@@ -0,0 +1 @@
+Integreat
```

Comment:
Sharing these texts between android and ios is probably not possible?

Reply:
Probably not, usage is maybe also different.